docs/dagsterVersions.json

```diff
@@ -0,0 +1,6 @@
+{
+  "1.9.13": "https://legacy-docs.dagster.io",
```

Comment:
These are placeholder links; once the versioned docs are created, we can update with the actual links.